jep/8/README.adoc

```diff
@@ -109,8 +107,7 @@ open-source conferences like FOSDEM, etc.
 ** Project expenses should happen within the GSoC timeframe between start of Community Bonding and end of the last coding phase
 ** GSoC budget may not be used to cover generic equipment costs: laptop, IDEs, internet, etc.
 ** GSoC budget may not be used to cover software license costs.
-The most of the projects are expected to be doable with only open-source or freeware software,
-otherwise it is up to the students and mentors to get the setup.
+Should a project require access to non-free resources (software or hardware), the project mentors and student are expected to obtain the resources at no cost to the GSoC budget.
```

Comment:
I propose to discuss it. Some requests may be viable (e.g. SaaS), so I would permit discussing options if such question arises. Org admins may help to reach out to potential sponsors

Reply:
```suggestion
Should a project require access to non-free resources (software or hardware), the project mentors and student are expected to discuss options with org admins. Org admins may help to reach out to potential sponsors.
```